tools/scripts/ansible/requirements.yml

```diff
@@ -1,3 +1,13 @@
-sprylab.tomcat-ubuntu
-geerlingguy.mysql
-derjd.journald
+- src: geerlingguy.mysql
+  version: 3.3.0
+- src: geerlingguy.nginx
+  version: 2.8.0
+- src: derjd.journald
+  version: 0.0.1
+- src: robertdebock.tomcat
+  version: 4.6.2
+# The following are also required for robertdebock.tomcat
+- src: robertdebock.bootstrap
```

Comment:
can we avoid adding these manually, or test whether they're actually used/needed as dependencies?

Reply:
@btangmu is it a bug in robertdebock.tomcat? maybe file a bug upstream then

Reply:
probably not a bug just my lack of understanding